Assets/LeapMotion+OVR/SystemWipe/QuickSwitcher.cs

```diff
@@ -145,7 +124,7 @@ private enum TransitionState { ON, OFF, MANUAL, TWEENING };
 	}
 
 	private IEnumerator doPositionTween(float goalPercent, float transitionTime, TweenCompleteDelegate onComplete = null) {
-//		Debug.Log("doPositionTween: " + goalPercent);
+    Debug.Log("doPositionTween: " + goalPercent);
```

Comment:
Debug here is outputting whenever quick switch is triggered.  Probably should be commented out or otherwise disabled.